style.css

```diff
@@ -107,3 +110,357 @@ body {
 .fab {
   color: #505f79;
 }
+
+/* works section starts here */
+
+.works-section-container {
+  background-color: #e5e5e5;
+  width: 100%;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  margin-top: 114px;
+  display: grid;
+  grid-template-rows: 552px 552px 552px 552px;
+  row-gap: 88px;
+}
```

Comment:
- [ ] You're using grid here incorrectly, you shouldn't give a fixed size to a row, instead, you can do this:

```
.works-section-container {
  background-color: #e5e5e5;
  width: 100%;
  border: 1px solid #dfe1e6;
  border-radius: 16px;
  margin-top: 114px;
  display: grid;
  row-gap: 88px;
  place-items: center;
}
```

This will let the cards be contained and centered in the grid container, so please change that accordingly. 😉

It will be like this:

![image](https://user-images.githubusercontent.com/20567503/177771738-be4ae6f7-7608-4e2f-b862-4ee0f297f897.png)

Note: please know that this will only work if you remove the fixed `width` from your body as I mentioned above.